web/src/features/preview/components/OutputScreen.tsx

```
@@ -31,43 +34,121 @@ export const OutputScreen: React.FC<OutputScreenProps> = ({ onBack, isGenerating
+                </div>
+
+                <div className="shrink-0 px-2 pb-2">
+                    <div className="grid grid-cols-2 gap-1 rounded-2xl border border-white/10 bg-[#1F1F1F] p-1">
+                        <button
+                            onClick={() => setMobileActiveTab('chat')}
+                            className={`rounded-xl px-3 py-2 text-sm font-medium transition-colors ${
+                                mobileActiveTab === 'chat'
+                                    ? 'bg-[#2B2B2B] text-white'
+                                    : 'text-[#91908F] hover:text-white hover:bg-white/5'
+                            }`}
+                        >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)